src/core/src/manifest.rs

```diff
@@ -329,9 +332,12 @@ mod test {
     use tempfile::TempDir;
 
     use super::Manifest;
+    use crate::collection::Collection;
+    use crate::encodings::HashFunctions;
+    use crate::selection::{Select, Selection};
 
     #[test]
-    fn manifest_from_pathlist() {
+    fn test_manifest_from_pathlist() {
```

Comment:
Test discovery in rust is done by the `#[test]` attribute, so adding `test_` to all test names doesn't add much value (other than makes names longer when running `cargo test`, but since you're running `cargo test` they are... tests already =])
```suggestion
    fn manifest_from_pathlist() {
```

Reply:
sure, just was in a py mindset 😂